README
NAME
    Dist::Zilla::Plugin::SpellingCommonMistakesTests - Release tests for
    common POD spelling mistakes

VERSION
    version 1.000001

SYNOPSIS
    In your dist.ini file:

        [SpellingCommonMistakesTests]

DESCRIPTION
    This is an extension of Dist::Zilla::Plugin::InlineFiles, providing the
    following file:

        xt/release/common_spelling.t - a standard Pod::Spell::CommonMistakes test

SEE ALSO
    Please see those modules/websites for more information related to this
    module.

    *   Pod::Spell::CommonMistakes

    *   Test::Pod::Spelling::CommonMistakes

INSTALLATION
    See perlmodinstall for information and options on installing Perl
    modules.

AUTHOR
    Adam Lesperance <lespea@gmail.com>

SUPPORT
  Perldoc
    You can find documentation for this module with the perldoc command.

      perldoc Dist::Zilla::Plugin::SpellingCommonMistakesTests

  Websites
    The following websites have more information about this module, and may
    be of help to you. As always, in addition to those websites please use
    your favorite search engine to discover more resources.

    *   Search CPAN

        <http://search.cpan.org/dist/Dist-Zilla-Plugin-SpellingCommonMistake
        sTests>

    *   RT: CPAN's Bug Tracker

        <http://rt.cpan.org/NoAuth/Bugs.html?Dist=Dist-Zilla-Plugin-Spelling
        CommonMistakesTests>

    *   AnnoCPAN: Annotated CPAN documentation

        <http://annocpan.org/dist/Dist-Zilla-Plugin-SpellingCommonMistakesTe
        sts>

    *   CPAN Ratings

        <http://cpanratings.perl.org/d/Dist-Zilla-Plugin-SpellingCommonMista
        kesTests>

    *   CPAN Forum

        <http://cpanforum.com/dist/Dist-Zilla-Plugin-SpellingCommonMistakesT
        ests>

    *   CPANTS Kwalitee

        <http://cpants.perl.org/dist/overview/Dist-Zilla-Plugin-SpellingComm
        onMistakesTests>

    *   CPAN Testers Results

        <http://cpantesters.org/distro/D/Dist-Zilla-Plugin-SpellingCommonMis
        takesTests.html>

    *   CPAN Testers Matrix

        <http://matrix.cpantesters.org/?dist=Dist-Zilla-Plugin-SpellingCommo
        nMistakesTests>

  Bugs / Feature Requests
    Please report any bugs or feature requests by email to
    "bug-dist-zilla-plugin-spellingcommonmistakestests at rt.cpan.org", or
    through the web interface at
    <http://rt.cpan.org/NoAuth/ReportBug.html?Queue=Dist-Zilla-Plugin-Spelli
    ngCommonMistakesTests>. You will be automatically notified of any
    progress on the request by the system.

  Source Code
    The code is open to the world, and available for you to hack on. Please
    feel free to browse it and play with it, or whatever. If you want to
    contribute patches, please send me a diff or prod me to pull from your
    repository :)

    <https://github.com/lespea/dist-zilla-plugin-spellingcommonmistakestests
    >

      git clone git://github.com/lespea/dist-zilla-plugin-spellingcommonmistakestests.git

COPYRIGHT AND LICENSE
    This software is copyright (c) 2011 by Adam Lesperance.

    This is free software; you can redistribute it and/or modify it under
    the same terms as the Perl 5 programming language system itself.

DISCLAIMER OF WARRANTY
    BECAUSE THIS SOFTWARE IS LICENSED FREE OF CHARGE, THERE IS NO WARRANTY
    FOR THE SOFTWARE, TO THE EXTENT PERMITTED BY APPLICABLE LAW. EXCEPT WHEN
    OTHERWISE STATED IN WRITING THE COPYRIGHT HOLDERS AND/OR OTHER PARTIES
    PROVIDE THE SOFTWARE "AS IS" WITHOUT WARRANTY OF ANY KIND, EITHER
    EXPRESSED OR IMPLIED, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
    WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE. THE
    ENTIRE RISK AS TO THE QUALITY AND PERFORMANCE OF THE SOFTWARE IS WITH
    YOU. SHOULD THE SOFTWARE PROVE DEFECTIVE, YOU ASSUME THE COST OF ALL
    NECESSARY SERVICING, REPAIR, OR CORRECTION.

    IN NO EVENT UNLESS REQUIRED BY APPLICABLE LAW OR AGREED TO IN WRITING
    WILL ANY COPYRIGHT HOLDER, OR ANY OTHER PARTY WHO MAY MODIFY AND/OR
    REDISTRIBUTE THE SOFTWARE AS PERMITTED BY THE ABOVE LICENCE, BE LIABLE
    TO YOU FOR DAMAGES, INCLUDING ANY GENERAL, SPECIAL, INCIDENTAL, OR
    CONSEQUENTIAL DAMAGES ARISING OUT OF THE USE OR INABILITY TO USE THE
    SOFTWARE (INCLUDING BUT NOT LIMITED TO LOSS OF DATA OR DATA BEING
    RENDERED INACCURATE OR LOSSES SUSTAINED BY YOU OR THIRD PARTIES OR A
    FAILURE OF THE SOFTWARE TO OPERATE WITH ANY OTHER SOFTWARE), EVEN IF
    SUCH HOLDER OR OTHER PARTY HAS BEEN ADVISED OF THE POSSIBILITY OF SUCH
    DAMAGES.



Changes
Revision history for Dist-Zilla-Plugin-SpellingCommonMistakesTests

1.000001  2011-03-17 06:12:34 America/Chicago
    [BugFix]
    *   Correct dist.ini name in the synopsis


1.000000  2011-03-17 06:01:42 America/Chicago
    *   Initial version


Build.PL

use strict;
use warnings;

use Module::Build 0.3601;


my %module_build_args = (
  'build_requires' => {
    'Dist::Zilla::Plugin::Bootstrap::lib' => '0',
    'English' => '0',
    'File::Find' => '0',
    'File::Temp' => '0',
    'Module::Build' => '0.3601',
    'Test::More' => '0.88'
  },
  'configure_requires' => {
    'Module::Build' => '0.3601'
  },
  'dist_abstract' => 'Release tests for common POD spelling mistakes',
  'dist_author' => [
    'Adam Lesperance <lespea@gmail.com>'
  ],
  'dist_name' => 'Dist-Zilla-Plugin-SpellingCommonMistakesTests',
  'dist_version' => '1.000001',
  'license' => 'perl',
  'module_name' => 'Dist::Zilla::Plugin::SpellingCommonMistakesTests',
  'recommends' => {},
  'recursive_test_files' => 1,
  'requires' => {
    'Dist::Zilla::Plugin::InlineFiles' => '0',
    'Moose' => '0',
    'Pod::Spell::CommonMistakes' => '0',
    'perl' => '5.008',
    'utf8' => '0'
  },
  'script_files' => []
);


my $build = Module::Build->new(%module_build_args);

$build->create_build_script;


Makefile.PL

use strict;
use warnings;

BEGIN { require 5.008; }

use ExtUtils::MakeMaker 6.31;



my %WriteMakefileArgs = (
  'ABSTRACT' => 'Release tests for common POD spelling mistakes',
  'AUTHOR' => 'Adam Lesperance <lespea@gmail.com>',
  'BUILD_REQUIRES' => {
    'Dist::Zilla::Plugin::Bootstrap::lib' => '0',
    'English' => '0',
    'File::Find' => '0',
    'File::Temp' => '0',
    'ExtUtils::MakeMaker' => '6.31',
    'Test::More' => '0.88'
  },
  'CONFIGURE_REQUIRES' => {
    'ExtUtils::MakeMaker' => '6.31'
  },
  'DISTNAME' => 'Dist-Zilla-Plugin-SpellingCommonMistakesTests',
  'EXE_FILES' => [],
  'LICENSE' => 'perl',
  'NAME' => 'Dist::Zilla::Plugin::SpellingCommonMistakesTests',
  'PREREQ_PM' => {
    'Dist::Zilla::Plugin::InlineFiles' => '0',
    'Moose' => '0',
    'Pod::Spell::CommonMistakes' => '0',
    'utf8' => '0'
  },
  'VERSION' => '1.000001',
  'test' => {
    'TESTS' => 't/*.t'
  }
);


unless ( eval { ExtUtils::MakeMaker->VERSION(6.56) } ) {
  my $br = delete $WriteMakefileArgs{BUILD_REQUIRES};
  my $pp = $WriteMakefileArgs{PREREQ_PM};
  for my $mod ( keys %$br ) {
    if ( exists $pp->{$mod} ) {
      $pp->{$mod} = $br->{$mod} if $br->{$mod} > $pp->{$mod};
    }
    else {
      $pp->{$mod} = $br->{$mod};
    }
  }
}

delete $WriteMakefileArgs{CONFIGURE_REQUIRES}
  unless eval { ExtUtils::MakeMaker->VERSION(6.52) };

WriteMakefile(%WriteMakefileArgs);



